crates/rolldown/src/stages/link_stage/bind_imports_and_exports.rs

```diff
@@ -619,10 +620,25 @@ impl BindImportsAndExportsContext<'_> {
           Specifier::Star => {
             MatchImportKind::Namespace { namespace_ref: importer_record.namespace_ref }
           }
-          Specifier::Literal(alias) => MatchImportKind::NormalAndNamespace {
-            namespace_ref: importer_record.namespace_ref,
-            alias: alias.clone(),
-          },
+          Specifier::Literal(alias) => {
+            if alias.as_str() == "default" {
+              // `ExternalModule` has no `ExportsKind`, here the module must be `NormalModule`
+              let module = self.index_modules[tracker.importee].as_normal().unwrap();
+              if module.ast_usage.contains(EcmaModuleAstUsage::ModuleRef) {
```

Comment:
It's not safe, because the importee module  commonjs maybe contains `__esModule` flag.

Reply:
Can you provide a example?

Reply:
I got it.